verification/cbmc_stubs/math.c

```
@@ -2087,12 +2087,12 @@ long double remainderl(long double x, long double y) { return __sort_of_CPROVER_
 #define __CPROVER_MATH_H_INCLUDED
 #endif
 
-double fabs (double d);
+double avocado_fabs (double d);
 
-double copysign(double x, double y)
+double avocado_copysign(double x, double y)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'avocado_copysign' of similar type ('double') are easily swapped by mistake

(bugprone-easily-swappable-parameters)